File: q1.py
#1 - Faça um código em python3 que leia o arquivo data.csv;

import csv

#Abertura do arquivo de dados na var dados
with open('src/data.csv') as dados:
    
    #Configurando o signo de tabulacao
    leitor = csv.DictReader(dados, delimiter=',')
    
    #Inicia o iterator na primeira linha [cabecario] [dessa forma pula-se a linha com as definicoes de coluna]
    leitor.__next__()
    
    #Iterator para percorrer o arquivo de dados
    for row in leitor:
        print( row )

File: q1.java
import java.util.*;

//1 - Faça um código em python3 que leia o arquivo data.csv;

public class Teste{
	public static void main(String []args){
		//Endereco do arquivo	
		String end = (new FileReader ("data (1).csv");
		try{
			//Buffer de leitura	abertura		
			BufferedReader arquivo = new BufferedReader(new FileReader(end));
			while(arquivo.ready()){
				String linha = arquivo.readLine();
				System.out.println(linha+"\n");
			}
			
			arquivo.close();
		}catch(IOException ioe){
			ioe.printStackTrace();
		} 
	}
}